cmd/sync/aws_test.go

```diff
@@ -161,3 +163,26 @@ func areEqualUpstreamsAWS(u1 awsUpstream, u2 Upstream) bool {
 
 	return true
 }
+
+func TestPrepareBatches(t *testing.T) {
+	const maxItems = 3
+	ids := []string{"i-394ujfs", "i-dfdinf", "i-fsfsf", "i-8hr83hfwif", "i-nsnsnan"}
+	instanceIds := make([]*string, len(ids))
+
+	for i := 0; i < len(ids); i++ {
+		instanceIds[i] = &ids[i]
+	}
+
+	batches := prepareBatches(maxItems, instanceIds)
+
+	if len(batches) > len(ids)/maxItems+1 {
```

Comment:
because we have a specific input here, perhaps it makes sense to use a number, like `len(batches) != 2`, so that is is easier to read the test?